src/app/features/loader/loader.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, OnInit } from '@angular/core';
+import { LoaderService } from '@core/services/loader/loader.service';
+
```

Comment:
Loader will be using through whole application, so we could move this component into 'shared' folder